packages/block-library/src/details/save.js

```diff
@@ -1,14 +1,19 @@
 /**
  * WordPress dependencies
  */
-import { useBlockProps, InnerBlocks } from '@wordpress/block-editor';
+import { RichText, useBlockProps, InnerBlocks } from '@wordpress/block-editor';
+import { __ } from '@wordpress/i18n';
```

Comment:
We can't use translations in save.js

https://github.com/WordPress/gutenberg/pull/45055/files/731a8e802bc2b0c7ba64c8fc1ef0bdaf91a0eee9#r1144295804